notebooks/1.4-fp-graph-communities.ipynb

```diff
@@ -0,0 +1,2348 @@
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 328.066406 340.121094 C 327.417969 288.058594 351.988281 254.429688 401.785156 239.226562 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 246.953125 247.292969 C 339.488281 219.628906 385.433594 157.503906 384.78125 60.925781 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 323.730469 476.871094 L 486.210938 381.535156 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 246.953125 247.292969 C 246.304688 150.710938 292.25 88.589844 384.78125 60.925781 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 307.761719 263.078125 L 347.8125 194.019531 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 379.011719 536.703125 C 317.976562 530.132812 282.066406 496.632812 271.285156 436.199219 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 477.75 313.859375 L 271.285156 436.199219 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 246.953125 247.292969 L 384.78125 60.925781 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 319.300781 433.277344 C 297.367188 449.484375 286.300781 471.222656 286.105469 498.492188 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M 319.300781 433.277344 C 319.105469 460.550781 308.039062 482.289062 286.105469 498.492188 \"/>\n",
+       "<path style=\"fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke:rgb(26.666667%,26.666667%,26.666667%);stroke-opacity:1;stroke-miterlimit:10;\" d=\"M
```

Comment:
Is it also possible to visualize this graph? I guess this one already includes knowledge about detected communities?

Reply:
I have not looked into it further since the dendrogram plot was not working for me so far.